typescript/sdk/src/ism/EvmIsmModule.ts

```diff
@@ -466,14 +466,14 @@ export class EvmIsmModule extends HyperlaneModule<
     config.domains = availableDomains;
 
     // deploy the submodules first
-    const submoduleAddresses: Address[] = await Promise.all(
-      Object.keys(config.domains).map(async (origin) => {
-        const { address } = await this.deploy({
-          config: config.domains[origin],
-        });
-        return address;
-      }),
-    );
+    const submoduleAddresses: Address[] = [];
```

Comment:
curious what wasn't working and why changing promise.all to for loops fixes things

Reply:
Promise.all is not a good idea, because concurrency support in signers is very poor, so you'll end up signing multiple transactions with the same nonce

Reply:
(not a good idea for a single chain and single signer)